legal-api/report-templates/template-parts/special-resolution/resolutionApplication.html

```diff
@@ -0,0 +1,73 @@
+{% if toLegalName %}
+<div class="no-page-break">
+  <div class="separator mt-4"></div>
+  <div class="section-title mt-4">Change of Business Name</div>
+</div>
+<table class="section-data business-change-table mt-4" role="presentation">
+    <tr class="no-page-break">
+        <td class="col-50">
+            <div class="section-sub-title">From</div>
+            <div class="{{'upper-text' if fromLegalName!='Not Available'}} pt-2">{{ fromLegalName }}</div>
+        </td>
+        <td class="col-50">
+            <div class="section-sub-title">To</div>
+            <div class="{{'upper-text' if toLegalName!='Not Available'}} pt-2">{{ toLegalName }}</div>
+        </td>
+    </tr>
+    <tr>
+        <td class="section-data" colspan="2">
+            <div class="section-sub-title">Name Request Number</div>
+            <div class="{{'upper-text' if nrNumber!='Not Available'}} pt-2">{{ nrNumber }}</div>
+        </td>
+    </tr>
+</table>
+<div class="separator mt-4"></div>
+{% endif %}
+
+{% if prevCoopAssociationType %}
+<div class="no-page-break">
+  <div class="separator mt-4"></div>
+  <div class="section-title mt-4">Change of Association Type</div>
+</div>
+<table class="section-data business-change-table mt-4" role="presentation">
+    <tr class="no-page-break">
+        <td class="col-50">
+            <div class="section-sub-title">From</div>
+            <div class="{{'upper-text' if prevCoopAssociationType!='Not Available'}} pt-2">{{ prevCoopAssociationType }}</div>
+        </td>
+        <td class="col-50">
+            <div class="section-sub-title">To</div>
+            <div class="{{'upper-text' if newCoopAssociationType!='Not Available'}} pt-2">{{ newCoopAssociationType }}</div>
+        </td>
+    </tr>
+</table>
+<div class="separator mt-4"></div>
+{% endif %}
+
+{% if rulesInResolution %}
+<div class="no-page-break">
+    <div class="section-title mt-4">Rules</div>
+</div>
+<div class="section-data mt-4">A certified copy of the uploaded rules is attached with this filing.</div>
+<div class="separator mt-4"></div>
+{% endif %}
```

Comment:
"Changes will be described in the special resolution text." should be displayed instead of "A certified copy of the uploaded rules is attached with this filing." here.  

The scenario of when a pdf is uploaded for the rules still needs to be handled to display "A certified copy of the uploaded rules is attached with this filing."

Reply:
Hi @argush3 
`If rule change was filed in the special resolution, then show that row. If not, hide.`

I don't think `"Changes will be described in the special resolution text."` should be displayed.

Reply:
@kzdev420 That is not how I interpret the designs.  My interpretation is that if "rules" are not changed at all(no rules file uploaded and rules in resolution is not checked), then the rules section is not shown.

Otherwise, "Changes will be described in the special resolution text." or "A certified copy of the uploaded rules is attached with this filing." is shown depending on the option selected.

Please confirm with @ethantspitt if my interpretation is correct.

Reply:
Yup - Argus's comment is correct.

Reply:
Hi @argush3 @ethantspitt 
I just fixed it. pls review